services/opentsdb/service.go

```diff
@@ -42,6 +42,7 @@ const (
 	statBatchesTransmitFail      = "batchesTxFail"
 	statConnectionsActive        = "connsActive"
 	statConnectionsHandled       = "connsHandled"
+	statDroppedPoints            = "droppedPoints"
```

Comment:
Again, let's put the word invalid in there to give us and our users a hint about why the points were dropped.